tools/bazel.rc

```diff
@@ -0,0 +1,36 @@
+common:clang --repo_env=CC=clang-15
+common:clang --repo_env=CXX=clang++-15
+build:clang --action_env=CC=clang-15
+build:clang --action_env=CXX=clang++-15
+build:clang --host_action_env=CC=clang-15
+build:clang --host_action_env=CXX=clang++-15
+
+### ASan build. Clang only. ###
```

Comment:
nit: consider adding that both `--config asan --config clang` need to be passed to bazel